alg/teca_model_segmentation.py

```diff
@@ -0,0 +1,109 @@
+import sys
+import teca_py
+import numpy as np
+
+class teca_model_segmentation:
+    """
+    Computes summary statistics, histograms on sorted, classified,
+    TC trajectory output.
```

Comment:
Note that this is detecting atmospheric rivers (ARs), not tropical cyclones (TCs).

I'd suggest changing the docstring to: `Given an input field of integrated vapor transport, calculates the probability of AR presence at each gridcell`